src/test/java/org/mule/extension/db/integration/model/OracleTestDatabase.java

```diff
@@ -158,6 +158,23 @@ public void createStoredProcedureAddOne(DataSource dataSource) throws SQLExcepti
     createStoredProcedure(dataSource, sql);
   }
 
+  @Override
+  public void createStoredProcedureAddOneNoSchema(DataSource dataSource) throws SQLException {
+    String sql = "CREATE OR REPLACE PROCEDURE addOne(num IN OUT INTEGER) AS\n" +
+        "BEGIN\n" +
+        "   num := num + 1;\n" +
+        "END;";
+
+    createStoredProcedure(dataSource, sql);
+
+    sql = "CREATE OR REPLACE PROCEDURE mathFunction.addOne(num IN OUT INTEGER) AS\n" +
```

Comment:
Same, why are you creating this SP here? There is already a method to create `mathFunction.addOne`